Upated Japanese translation

tkusano opened the pull request:
Deleted obsoleted messages which are mostly related to old onboarding messages.

tkusano commented:
- Translate new messages
-  Replace 3 dots with HORIZONTAL ELLIPSIS

tkusano commented:
@Hima-Zinn @dolciss @oboenikui @noritada @middlingphys @hibiki @reindex-ot @haoyayoi @vyv03354

Please review

dolciss commented:
Thanks for the latest translation.
LGTM👍

Hima-Zinn reviewed (commented):
Thank you for your work.

tkusano commented:
Add new translations mainly related to https://github.com/bluesky-social/social-app/commit/3ece21cb45e8b74795f4eac33a1551b303196946 . Translated as follows.

deactivation -> 無効化
reactivation -> 再有効化

Changed 'マイアカウント' to 'アカウント' to simplify UI.

Also, changed half-width '()' to full-width '（）'.

dolciss commented on src/locale/locales/ja/messages.po:
```diff
@@ -5089,9 +5043,9 @@ msgstr "サポートフォームは移動しました。サポートが必要な
-msgstr "試せるフィードはたくさんあります："
+#: src/screens/Settings/components/DeactivateAccountDialog.tsx:35
+msgid "There is no time limit for account deactivation, come back any time."
+msgstr "アカウントの無効化に期限はありません。いつでも戻ってこれます。"
```
```suggestion
msgstr "アカウントの無効化に期限はありません。いつでも戻ってこられます。"
```

tkusano replied:
fixed in https://github.com/bluesky-social/social-app/pull/4477/commits/fa74fb9dfef6bdeeaf24c589ac52b0f221c51319 . thanks.

dolciss reviewed (commented):
Thanks for the additional translation.[^1]
I would appreciate it if you could correct one minor point.

[^1]: 別で提案した「無効化」案を採用いただきありがとうございます。その際にあった復活できないinvalidate的な意味に取られる可能性も、UIに復活できる旨の説明があるため大丈夫かと私も考えています。

pfrazee reviewed (approved):
Thanks!!
